patches/chrome-browser-devtools-devtools_ui_bindings.cc.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/chrome/browser/devtools/devtools_ui_bindings.cc b/chrome/browser/devtools/devtools_ui_bindings.cc
+index e42fa03dc0a29491ff652ec0cb68e14bbc0661a3..2289b77e1d86c910b03534e9ab1f56ea35ba8d6a 100644
+--- a/chrome/browser/devtools/devtools_ui_bindings.cc
++++ b/chrome/browser/devtools/devtools_ui_bindings.cc
+@@ -1761,6 +1761,9 @@ void DevToolsUIBindings::AddDevToolsExtensionsToClient() {
+                    base::Value(std::move(forbidden_origins)));
+   CallClientMethod("DevToolsAPI", "addExtensions",
+                    base::Value(std::move(results)));
++#if !defined(OFFICIAL_BUILD)
```

Comment:
pls replace this with define `BRAVE_DEV_TOOLS_UI_BINDINGS_ADD_DEV_TOOLS_EXTENSIONS_TO_CLIENT` and add `chromium_src/..` file with a declaration.

Reply:
also this thing maybe worth upstreaming.